src/components/molecules/MessageCard.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="bg-white p-6 space-y-3" >
+        <span class="uppercase text-sm font-light font-body">{{ timestamp }}</span>
+        <h1 class="font-title text-2xl font-semibold">{{ title }}</h1>
+        
+        <!-- <p class="font-normal text-lg" v:for="paragraph of paragraphs">
+            {{ paragraph }}
+        </p> -->
```

Comment:
Clean up the comments if they are not needed :) Or leave a comment explaining what they are used for.